README.md

```diff
@@ -1,76 +1,86 @@
+-	[Terraform](https://www.terraform.io/downloads.html) 0.12.x or higher
+-	[Go](https://golang.org/doc/install) 1.19 or higher to build the provider plugin
 
-Building The Provider
----------------------
+## Build the provider
 
-Clone repository to: `$GOPATH/src/github.com/splunk-terraform/terraform-provider-signalfx`
+To build the provider, follow these steps:
 
-```sh
-$ git clone git@github.com:splunk-terraform/terraform-provider-signalfx.git $GOPATH/src/github.com/splunk-terraform/terraform-provider-signalfx
-```
+1. Clone the repository to: `$GOPATH/src/github.com/splunk-terraform/terraform-provider-signalfx`:
 
-Enter the provider directory and build the provider
+   ```sh
+   $ git clone git@github.com:splunk-terraform/terraform-provider-signalfx.git $GOPATH/src/github.com/splunk-terraform/terraform-provider-signalfx
+   ```
 
-```sh
-$ cd $GOPATH/src/github.com/splunk-terraform/terraform-provider-signalfx
-$ make build
-```
+1. Enter the provider directory and build the provider:
 
-Using the provider
-----------------------
-If you're building the provider, follow the instructions to [install it as a plugin.](https://www.terraform.io/docs/plugins/basics.html#installing-a-plugin) After placing it into your plugins directory,  run `terraform init` to initialize it.
+   ```sh
+   $ cd $GOPATH/src/github.com/splunk-terraform/terraform-provider-signalfx
+   $ make build
+   ```
 
-Further [usage documentation is available on the Terraform website](https://www.terraform.io/docs/providers/signalfx/index.html).
+## Use the provider
 
-Developing the Provider
----------------------------
+If you're building the provider, follow the instructions to [install it as a plugin.](https://www.terraform.io/docs/plugins/basics.html#installing-a-plugin). After placing it into your plugins directory,  run `terraform init` to initialize it.
 
-If you wish to work on the provider, you'll first need [Go](http://www.golang.org) installed on your machine (version 1.11+ is *required*). You'll also need to correctly setup a [GOPATH](http://golang.org/doc/code.html#GOPATH), as well as adding `$GOPATH/bin` to your `$PATH`.
+Further [usage documentation](https://www.terraform.io/docs/providers/signalfx/index.html) is available on the Terraform website.
 
-To compile the provider, run `make build`. This will build the provider and put the provider binary in the `$GOPATH/bin` directory.
+## Develop the provider
 
-```sh
-$ make bin
-...
-$ $GOPATH/bin/terraform-provider-signalfx
-...
-```
+If you wish to work on the provider, you need the following:
 
-In order to test the provider, you can simply run `make test`.
+- [Go](http://www.golang.org) version 1.11 or higher
+- Configured [GOPATH](http://golang.org/doc/code.html#GOPATH)
+- `$GOPATH/bin` added to your `$PATH`
 
-```sh
-$ make test
-```
+To compile the provider, run `make build`. This builds the provider and put its binary inside the `$GOPATH/bin` directory:
+
+   ```sh
+   $ make bin
```

Comment:
I know this is not your change but it should say `make build`

Reply:
Fixed!